sites/all/modules/custom/demand_create/demand_create.module

```diff
@@ -0,0 +1,1632 @@
+    '#type' => 'item',
+    '#title' => 'Настройки нумерации',
+    '#value' => '',
+    '#prefix' => '<div class="ll">',
+    '#suffix' => '</div>',
+  );
+
+  $form['formula_number_type'] = array(
+    '#type' => 'select',
+    '#title' => 'Тип нумерации',
+    '#options' => array(
+      0 => '(Нет)',
+      1 => 'Сквозная',
+      2 => 'Привязка к разделам/подразделам',
+    ),
+    '#prefix' => '<div class="section-content"> <div class="container-inline">',
+    '#suffix' => '</div>',
+  );
+
+  $form['formula_number_view'] = array(
+    '#type' => 'radios',
+    '#attributes' => array('class' => array('n_visibl_r', 'num')),
+    '#title' => 'Вид нумерации',
+    '#default_value' => variable_get('align_text', 0),
+    '#options' => array(
+      0 => '<span></span>',
+      1 => '<span></span>',
+      2 => '<span></span>',
+      3 => '<span></span>',
+      4 => '<span></span>',
+      5 => '<span></span>',
+      6 => '<span></span>',
+      7 => '<span></span>',
+    ),
+  );
+
+  $form['formula_division_type'] = array(
+    '#type' => 'select',
+    '#title' => 'Вид разделителя между подписью и текстом:',
+    '#options' => array(
+      0 => '(Нет)',
+      1 => '.',
+      2 => ',',
+      3 => ')',
+      4 => '\\',
+      5 => '-',
+      6 => '*',
+      7 => '+',
+      8 => 'Пробел',
+    ),
+    '#prefix' => '<div class="container-inline">',
+    '#suffix' => '</div>',
+  );
+
+  $form['formula_number_setting_main_text'] = array(
+    '#type' => 'checkbox',
+    '#title' => 'Применить настройки основного текста',
+    '#suffix' => '</div>',
+  );
+
+  $form['item_lists'] = array(
+    '#type' => 'item',
+    '#title' => 'Списки',
+    '#theme_wrappers' => array(),
+    '#prefix' => '<h3>'.'Списки',
+    '#suffix' => '</h3>',
+    '#value' => '',
+  );
+
+  $form['item_formatting'] = array(
+    '#type' => 'item',
+    '#title' => 'Форматирование',
+    '#value' => '',
+    '#prefix' => '<div class="ll">',
+    '#suffix' => '</div>',
+  );
+
+  $form['indents_lists_level'] = array(
+    '#type' => 'select',
+    '#title' => 'Уровень списка:',
+    '#options' => array(
+      0 => 'S1',
+      1 => 'S2',
+      2 => 'S3',
+      3 => 'S4',
+      4 => 'S5',
+      5 => 'S6',
+    ),
+    '#prefix' => '<div class="section-content">',
+  );
+
+  $form['indents_lists_fields'] = array(
+    '#type' => 'textfield',
+    '#title' => 'Отступы от полей:',
+    '#size' => 10,
+    '#prefix' => '<div class="container-inline">',
+    '#suffix' => '</div>',
+  );
+
+  $form['indents_lists_number'] = array(
+    '#type' => 'textfield',
+    '#title' => 'Отступы от нумерации:',
+    '#size' => 10,
+    '#prefix' => '<div class="container-inline">',
+   '#suffix' => '</div>',
+  );
+
+  $form['lists_setting_main_text'] = array(
+    '#type' => 'checkbox',
+    '#title' => 'Применить настройки основного текста',
+  );
+
+  $form['item_valid_symbols'] = array(
+    '#type' => 'item',
+    '#title' => 'Допустимые символы и знаки'),
```

Comment:
Parse error: syntax error, unexpected ',' in /Users/stanislavmixnovich/Sites/devdesktop/GOST/sites/all/modules/custom/demand_create/demand_create.module on line 1487